docs/resources/Channel.md

```diff
@@ -21,7 +21,7 @@ Represents a guild or DM channel within Discord.
 | user_limit?                    | integer                                                                    | the user limit of the voice channel                                                                                                                                                             |
 | rate_limit_per_user?\*         | integer                                                                    | amount of seconds a user has to wait before sending another message (0-21600); bots, as well as users with the permission `manage_messages` or `manage_channel`, are unaffected                 |
 | recipients?                    | array of [user](#DOCS_RESOURCES_USER/user-object) objects                  | the recipients of the DM                                                                                                                                                                        |
-| icon?                          | ?string                                                                    | icon hash                                                                                                                                                                                       |
+| icon?                          | ?string                                                                    | icon hash (for group DMs)                                                                                                                                                                                      |
```

Comment:
```suggestion
| icon?                          | ?string                                                                    | icon hash (for group DMs)                                                                                                                                                                                      |
```

I would say it's better to write "icon hash of the group DM"

That's how it is done in the other columns.